### PR TITLE
Warn when browser version is ran in node

### PR DESCRIPTION
**Pull request by pornel**

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -460,6 +460,12 @@ Libraries like [co](https://github.com/tj/co) or a web framework like [koa](http
 Note that superagent expects the global `Promise` object to be present. You'll need a polyfill to use promises in Internet Explorer or Node.js 0.10.
 
 
-## Using browser version in electron
+## Browser and node versions
 
-[Electron](http://electron.atom.io/) developers report if you would prefer to use the browser version of superagent instead of the node version, you can `require('superagent/superagent')`. Your requests will now show up in the chrome developer tools network tab. Note this environment is not covered by automated test suite and not officially supported.
+Superagent has two implementations: one for web browsers (using XHR) and one for Node.JS (using request module). By default Browserify and WebPack will pick the browser version.
```
Comment (focusaurus) [inline docs/index.md]: Using node core `http`, not the npm `request` module, right?

Comment (pornel) [inline docs/index.md]: Yup

**Comment by gcarol** [comment]
H

**Review by gcarol — COMMENTED** [review]
Y

**Pull request merged**